cms-adapter/src/Rsbc.Dmf.CaseManagement/Manager/DocumentManager.cs

```diff
@@ -277,6 +278,72 @@ public Document GetDmer(Guid caseId)
                 .FirstOrDefault();
 
             return _mapper.Map<Document>(document);
-        }   
+        }
+
+        /// <summary>
```

Comment:
personally, the xml documents are for old documentation tools that no longer exist or are being used. I find they add more confusion, so I generally just delete them. This one is already out of date, for example. Which is generally what happens with XML automated comments